tests/phpunit/TestFramework/Coverage/XmlReport/TestLocatorTest.php

```diff
@@ -126,26 +126,6 @@ public function rangeProvider(): iterable
                     'testFilePath' => '/path/to/acme/FooTest.php',
                     'testExecutionTime' => 0.456,
                 ],
-                [
-                    'testMethod' => 'Infection\Acme\FooTest::test_it_can_do_0',
-                    'testFilePath' => '/path/to/acme/FooTest.php',
-                    'testExecutionTime' => 0.123,
-                ],
-                [
-                    'testMethod' => 'Infection\Acme\FooTest::test_it_can_do_1',
-                    'testFilePath' => '/path/to/acme/FooTest.php',
-                    'testExecutionTime' => 0.456,
-                ],
-                [
-                    'testMethod' => 'Infection\Acme\FooTest::test_it_can_do_1',
-                    'testFilePath' => '/path/to/acme/FooTest.php',
-                    'testExecutionTime' => 0.456,
-                ],
-                [
-                    'testMethod' => 'Infection\Acme\FooTest::test_it_can_do_0',
-                    'testFilePath' => '/path/to/acme/FooTest.php',
-                    'testExecutionTime' => 0.123,
-                ],
```

Comment:
better than 1000 words, here is a result of removing duplicates. This test was a proof that we had a bug :)

Now - there are no more duplicates (removed array values are duplicates)